src/run.sh

```diff
@@ -209,11 +209,10 @@ function submit(){
           if [[ ${6#\'} != /lscratch* ]]; then
             CLUSTER_OPTS="sbatch --cpus-per-task {cluster.threads} -p {cluster.partition} -t {cluster.time} --mem {cluster.mem} --job-name={params.rname} -e $SLURM_DIR/slurm-%j_{params.rname}.out -o $SLURM_DIR/slurm-%j_{params.rname}.out"
           fi
-          # Create sbacth script to build index
     cat << EOF > kickoff.sh
 #!/usr/bin/env bash
-#SBATCH --cpus-per-task=16 
-#SBATCH --mem=96g
+#SBATCH --cpus-per-task=16
+#SBATCH --mem=32g
 #SBATCH --time=5-00:00:00
```

Comment:
6 days for the wall time should be good. I only mention this because I see some rules with a 5 day wall time. I am not sure if they are currently being used though:
![image](https://github.com/OpenOmics/metamorph/assets/18038345/c45508fc-7f30-4634-80de-5db33e8804a3)

Reply:
Yeah we would have to profile to be certain, I think master job and assembly should probably be set to `*norm` max wall time.